api/main.py

```diff
@@ -7,7 +7,7 @@
 
 
 def main():
-    websocket_port = int(os.environ['API_WEBSOCKET_PORT'])
+    websocket_port = int(os.environ['PORT'])
```

Comment:
I like the long name better - if another port number was needed in the future this would need to change. This is our only port now, so I think it's fine, but I don't know why this change was made.

Reply:
heroku specifies the port you have to listen on with the PORT environment variable